src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -2725,7 +2723,7 @@ private static bool WinCreateSymbolicLink(string path, string strTargetPath, boo
             var flags = isDirectory ? NativeMethods.SymbolicLinkFlags.Directory : NativeMethods.SymbolicLinkFlags.File;
 
             Version minBuildOfDeveloperMode = new Version(10, 0, 14972, 0);
```

Comment:
`minBuildOfDeveloperMode` can now be removed.

Reply:
Removed.